frontend/www/js/omegaup/components/common/Navbar.test.ts

```diff
@@ -63,6 +63,14 @@ describe('Navbar.vue', () => {
     expect(wrapper.find('.navbar-right').text()).toBe(T.navLogIn);
   });
 
+  it('Should handle common navbar to logged user', () => {
+    const wrapper = mount(common_Navbar, {
+      propsData: { ...propsData },
```

Comment:
```suggestion
      propsData,
```

Así quedaría más simple.